packages/block-library/src/template-part/edit/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect } from '@wordpress/element';
+import { EntityProvider } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import useTemplatePartPost from './use-template-part-post';
+import TemplatePartInnerBlocks from './inner-blocks';
+import TemplatePartPlaceholder from './placeholder';
+
+export default function TemplatePartEdit( {
```

Comment:
Minor non-blocking: When we use the block navigator the blocks nested inside top-level template parts do not appear.
If we select a template part, the blocks nested inside the template part (including blocks nested inside a descendent template part) appear.
I guess even on the top-level view blocks nested inside a template part should appear.

Reply:
I think this is how the navigator works for all inner blocks. I get the same behavior with the Cover block.